MB056625/Auto.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Dziedziczenie;
+namespace MB056625
+{
+    class Auto : IPojazd
+    {
+        public Silnik Silnik { get; private set; }
+
+        public void Jedz() => Jedz(100);
+
+
+        public void Jedz(int dystans)
+        {
+
+            string a = "";
+            for (int i = 0; i < dystans / this.Silnik.Spalanie; i++)
+            {
+                Console.Clear();
```

Comment:
Proszę nie używać Console.Clear. W tym samym oknie zostaną wywołane prace innych studentów a to je usunie.